app/views/apps/_app.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+json.extract! app, :id, :name, :team_id, :created_at, :updated_at
```

Comment:
Is jbuilder the rails norm? DId we consider Active Model Serializers? Maybe this was also auto generated?

Reply:
Yes, this is auto generated when running rails generate scaffold App.